### ISSUE TITLE
Bind to "format document" key binding

### ISSUE THREAD

**Issue by penrillian-chrisa** [standard/vscode-standard#9]
Should this plugin automatically bind to the default "format document" key binding?

(It works just fine in chenxsan.vscode-standard-format)

**Comment by chenxsan** [comment]
No, I think it would be confusing when all those plugins automatically bind to the default `format document` key binding. So it won't be supported unless necessary.

**Comment by brucx** [comment]
Would you please consider making this configurable?

**Comment by chenxsan** [comment]
@brucx haha, I don't find any free time to implement this. You can submit a pull request if you like, I'm willing to merge it.

**Comment by falco467** [comment]
> Would you please consider making this configurable?

How would that be different from now? The user can just bind the Formatting Hotkey to JS Fix Problems and has the same effect !?

**Comment by mcdado** [comment]
https://code.visualstudio.com/blogs/2016/11/15/formatters-best-practices

The problem with creating a user shortcut is that is not easily shared between platforms (see Settings Sync) and other developers (shared projects). It would be much nicer to have a proper formatter.

**Comment by jurf** [comment]
> No, I think it would be confusing when all those plugins automatically bind to the default `format document` key binding. So it won't be supported unless necessary.

I do not understand. All the extensions I have used (Rust, C++, Java, Python) just plug in into the API – why should this extension be different? VSCode already provides settings to configure formatters per language.
